operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/ManagedInformerEventSource.java

```diff
@@ -27,7 +27,7 @@
 import io.javaoperatorsdk.operator.processing.event.source.*;
 
 @SuppressWarnings("rawtypes")
-public abstract class ManagedInformerEventSource<R extends HasMetadata, P extends HasMetadata, C extends ResourceConfiguration<R>>
+public abstract class ManagedInformerEventSource<R extends HasMetadata, P extends HasMetadata, C extends Informable<R>>
```

Comment:
(Unrelated: ManagedInformerEventSource is quite bad name, since we don't really have that notion.)

Reply:
It is an abstract class for event sources that use informers managed by JOSDK so it somewhat makes sense. What name would you propose instead?

Reply:
Yeah, not sure, we can keep it like this for now